src/command/mcl.zig

```diff
@@ -11,6 +11,10 @@ var line_accelerations: []u7 = undefined;
 const Direction = mcl.Direction;
 const Station = mcl.Station;
 
+const RegisterType = enum { x, y, wr, ww };
+var register_log_file: ?std.fs.File = null;
+var register_list: [4]?RegisterType = std.mem.zeroes([4]?RegisterType);
```

Comment:
Not sure about this data type and initialization. Would an [`EnumArray`](https://ziglang.org/documentation/master/std/#std.EnumArray) of booleans be a better fit?

Reply:
EnumArray might work as well. However, I did this for the sake of looping only to the provided register by user. If the user want to log only X and Y, the register list will look: [X,Y,null,null]. Then, we can use while loop to loop until 2nd element.

Reply:
Try to prefer type safety over micro optimizations like this. In a bounded guaranteed-length-4 array, the difference between 2 loop iterations and 4 loop iterations is meaningless. However, using EnumArray can prevent bugs such as duplicated identical registers in the array, accidental overwrites of existing registers in the array, etc.